### ISSUE TITLE
Carousel is not an actual carousel

### ISSUE THREAD

**Issue by pantonis** [primefaces/primeng#3896]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[X ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Carousel has a header and the buttons are in the header making it difficult to use. Very bad Ux. 99.9% of the carousels have buttons either in or on the sides of the main list and the scroller is usually on bottom. They dont have a header as well. 

**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**
Very very bad UX

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **PrimeNG version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by Ninja-Coding-Git** [comment]
i suggest use of [swiper](http://idangero.us/swiper/#.Wbf58MjyjIU) because **Gallery** and **Carousel** components are so old, unfriendly with: responsive, ux, touch devices.

**Comment by pantonis** [comment]
Any estimate?

**Comment by pantonis** [comment]
Any news on this one?

**Comment by rafa-as** [comment]
@cagataycivici we want to use your carousel component, but without touch support and without access to next previous functions are very weird to use

**Comment by cagataycivici** [comment]
Yes, a new Carousel is being worked on for 7.0, current one is outdated already and a new modern one is being designed.

**Comment by cagataycivici** [comment]
Postponed to 7.1

**Comment by pantonis** [comment]
what a shame

**Comment by pantonis** [comment]
I do not see it in 7,1 list

**Comment by nimbusparis** [comment]
This feature is not in 7.x or 8.x. Cancelled?

**Comment by cagataycivici** [comment]
It is done and will be available in 8.1.0.

![image](https://user-images.githubusercontent.com/686247/66306174-a85aa100-e909-11e9-89cc-b4a921f492b6.png)

**Comment by guiferrpereira** [comment]
@cagataycivici you can tell me when 8.1.0 will be available?
